docs/getting-started/products/yvaults/v3.md

```diff
@@ -0,0 +1,245 @@
+- **Debt Allocator:** Smart contract that incentivizes APY / debt allocation optimization by rewarding the best debt allocation (see [yStarkDebtAllocator](https://github.com/jmonteer/ystarkdebtallocator)).
+- **Strategy Staking Module:** Smart contract that allows players to sponsor specific strategies (so that they are added to the vault) by staking their YFI, making money if they do well and losing money if they don't.
+
+## Deployment
+We expect all the vaults available to be deployed from a Factory Contract, publicly available and callable. 
+
+Players deploying "branded" vaults (e.g. Yearn) will use a separate registry to allow permissioned endorsement of vaults for their product
+
+When deploying a new vault, it requires the following parameters:
+- **asset:** Address of the ERC20 token that can be deposited in the vault.
+- **name:** Name of Shares as described in ERC20.
+- **symbol:** Symbol of Shares ERC20.
+- **role_manager:** Account that can assign and revoke Roles.
+- **PROFIT_MAX_UNLOCK_TIME:** Max amount of time profit will be distributed over.
+
+## Normal Operation
+
+### Deposits / Mints
+Users can deposit ASSET tokens to receive yvTokens (SHARES).
+
+Deposits are limited under `depositLimit` and shutdown parameters. Read below for details.
+
+### Withdrawals / Redeems
+Users can redeem their shares at any time if there is liquidity available. 
+
+The redeem function will check if there are enough idle funds to serve the request. If there are not enough, it will revert. 
+
+Optionally, a user can specify a list of strategies to withdraw from. If a list of strategies is passed, the vault will try to withdraw from them.
+
+If not enough funds have been recovered to honor the full request, the transaction will revert.
+
+### Vault Shares
+Vault shares are ERC20 transferable yield-bearing tokens.
+
+They are ERC4626 compliant. Please read [ERC4626 compliance](https://hackmd.io/cOFvpyR-SxWArfthhLJb5g#ERC4626-compliance) to understand the implications. 
+
+### Accounting
+The vault will evaluate profit and losses from the strategies. 
+
+This is done by comparing the current debt of the strategy with the total assets the strategy is reporting to have. 
+
+- If `totalAssets < currentDebt`: the vault will record a loss
+- If `totalAssets > currentDebt`: the vault will record a profit
+
+Both loss and profit will impact the strategy's debt, increasing the debt (current debt + profit) if there are profits, and decreasing its debt (current debt - loss) if there are losses.
+
+#### Fees
+The Accountant module handles fee assessment and distribution. It will report the amount of fees that need to be charged, and the vault will issue shares for that amount of fees.
+
+#### Refunds
+Refunds are positive inflows for the vault, and are sent to the vault to reward it or compensate in anyway. The Accountant module will take care of them. 
```

Comment:
`The Accountant module will take care of them.` to
`The Accountant module handles refund accounting`, maybe